apps/Gruntfile.js

```diff
@@ -153,8 +153,16 @@ config.copy = {
       {
         expand: true,
         cwd: 'lib/droplet',
-        src: ['droplet-full' + droplet_suffix + '.js'],
-        dest: 'build/package/js/droplet/'
+        src: [
+          'droplet.js',
```

Comment:
Not 100% sure this is needed. It might just be that it's referenced from the source map. We had it in our /lib directory, so I figured I might as well include it along with the build. It's also possible that we don't actually want the .js.map unless we're in dev mode. Thoughts?